scanapp/models.py

```diff
@@ -129,39 +98,43 @@ def __str__(self):
 
```

Comment:
Start and end line should be best side-by-side, and try to keep the same order of the fields as they are in the ScanCode JSON.